snapcraft.yaml

```diff
@@ -12,6 +12,7 @@ description: |
 
 confinement: strict
 grade: stable
+base: core20
 
 apps:
     spread:
```

Comment:
adding an entry for personal-files to be able to easily use the google backend would be nice.

Reply:
Excellent suggestion, I think this would ideally be a followup?